src/port/port.tcl

```diff
@@ -2094,7 +2095,20 @@ proc action_info { action portlist opts } {
 
         # Spin through action options, emitting information for any found
         set fields {}
-        set opts_todo [array names options ports_info_*]
+
+        # This contains the display fields in random order
+        set opts_info [array names options ports_info_*]
+        # This contains all parameters in order given on command line
+        set opts_action $global_options(options_${action}_order)
+        # Get the display fields in order provided on command line
+        #  ::struct::set intersect does not keep order of items
```

Comment:
Maybe I am missing something in this comment, but are we actually using `struct::set` anywhere...?

Reply:
It is not used, I added the comment in case someone looks at this and thinks ::set intersect could be used (which is what I tried first).